services/dynamic-sidecar/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+All notable changes to this project will be documented in this file.
```

Comment:
MINOR: this "service", not project, right?  :-)
I personally like CHANGELOGs to keep a human-readable log of relevant changes but my experience is that this type of doc are easily forgotten ...